pyproject.toml

```diff
@@ -1,3 +1,8 @@
 [build-system]
 requires = ["setuptools >= 40.6.0", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+testpaths = [
+    "tests",
+]
```

Comment:
this is tripping Circle and for some reason pytest can't collect the tests, do we need it here? See https://github.com/ESMValGroup/ESMValTool/pull/2500 - such a shame that the 2500th PR/issue was a mere test :grin:

Reply:
Well, no. I added it because the `.pymon-journal` issue showed up again in this branch. (see https://github.com/ESMValGroup/ESMValTool/runs/4949630634?check_suite_focus=true). Clearly the fix we merged recently didn't fix it, so I thought I might limit the tests this way.

But since this really is an unrelated problem, I'll remove it here, re-open #2489, and then we can take that later.